src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1604,7 +1655,7 @@ ExitForFor:
                 Dim constraintClass = typeParameter.GetClassConstraint(useSiteDiagnostics)
                 If constraintClass IsNot Nothing Then
                     LookupInClass(result, constraintClass, name, arity, options, constraintClass, binder, useSiteDiagnostics)
-                    If result.StopFurtherLookup Then
+                    If result IsNot Nothing AndAlso result.StopFurtherLookup Then
```

Comment:
Nit: feels like all of these similar cases are perfect use cases for `result?.Property = True`

Reply:
> Nit: feels like all of these similar cases are perfect use cases for result?.Property = True

The comparison with true is not needed in VB  

---
In reply to: [184232418](https://github.com/dotnet/roslyn/pull/26398#discussion_r184232418) [](ancestors = 184232418)